platform-sdk/swirlds-config-processor/src/main/java/module-info.java

```diff
@@ -0,0 +1,11 @@
+module com.swirlds.config.processor {
+    provides javax.annotation.processing.Processor with
+            com.swirlds.config.processor.ConfigDataAnnotationProcessor;
+
+    requires static com.google.auto.service;
+    requires java.compiler;
+    requires transitive com.swirlds.config.api;
+    requires static com.github.spotbugs.annotations;
+    requires com.squareup.javapoet;
+    requires org.antlr.antlr4.runtime;
```

Comment:
Nit pick: Sort list by scopes:

```suggestion
    requires transitive com.swirlds.config.api;
    requires com.squareup.javapoet;
    requires java.compiler;
    requires org.antlr.antlr4.runtime;
    requires static com.google.auto.service;
    requires static com.github.spotbugs.annotations;
```